omnibus/package-scripts/agent/postinst

```diff
@@ -82,15 +82,13 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
             SYSTEMCTL_SKIP_SYSV=true systemctl enable $SERVICE_NAME || echo "[ WARNING ]\tCannot enable $SERVICE_NAME with systemctl"
             SYSTEMCTL_SKIP_SYSV=true systemctl enable $SERVICE_NAME-process || echo "[ WARNING ]\tCannot enable $SERVICE_NAME-process with systemctl"
             SYSTEMCTL_SKIP_SYSV=true systemctl enable $SERVICE_NAME-trace || echo "[ WARNING ]\tCannot enable $SERVICE_NAME-trace with systemctl"
-            SYSTEMCTL_SKIP_SYSV=true systemctl enable $SERVICE_NAME-security || echo "[ WARNING ]\tCannot enable $SERVICE_NAME-security with systemctl"
```

Comment:
I think you need to do equivalent changes in the `posttrans` file (used by the rpm package), to avoid enabling the service (+ remove the mention of the SysVinit file `/etc/init.d/datadog-agent-security`, which doesn't exist anymore).

I think the `datadog-agent-security` mentions in `preinst` and `prerm` can be left here, as they're about stopping an existing `datadog-agent-security` service before installing and before removing the package (which can happen if the service was manually enabled).